source/opt/aggressive_dead_code_elim_pass.cpp

```diff
@@ -269,7 +262,7 @@ void AggressiveDCEPass::AddBreaksAndContinuesToWorklist(
   get_def_use_mgr()->ForEachUser(contId, [&contId,
                                           this](ir::Instruction* user) {
     SpvOp op = user->opcode();
-    if (op == SpvOpBranchConditional) {
+    if (op == SpvOpBranchConditional || op == SpvOpSwitch) {
       // A conditional branch can only be a continue if it does not have a merge
```

Comment:
Recommend changing "conditional branch" to "conditional branch or switch"

Reply:
(It still takes me a long time to process this whole comment and algorithm. But I think it's ok.)